### PR TITLE
Lower number of queries

### PR DESCRIPTION
**Pull request by percona-csalguero**
The `GetDigestText` function is reading **THE SAME** table than `GetDigestRows` so, there is no reason to not to include the field `DIGEST_TEXT` in the query at line 114 https://github.com/percona/qan-agent/compare/perfschema-digest-text?expand=1#diff-c98c191a6a53ec5bd6e6595ae8605b79R114 and read all the needed info at once.

:warning: :warning: **We need to fix-update all the tests** :warning: :warning:

### CODE REVIEW

**Comment by arvenil** [comment]
@percona-csalguero Hey, why this never got merged? This change makes sense to me. I just fixed tests - looks good to be merged for me. If this still makes sense to you could you do full code review as I made some changes to test data and to one part of the code.

File: qan/analyzer/mysql/worker/perfschema/worker.go [hunk 1]
```diff
@@ -421,23 +401,12 @@ ROW_LOOP:
...
-					if err != nil {
-						w.logger.Error(err)
-						continue
-					}
+				digestText := row.DigestText
+				if classId == "2" && digestText == "" {
+					// To make explains works
+					digestText = `-- performance_schema.events_statements_summary_by_digest is full`
 				}
+
```
Comment (arvenil) [inline qan/analyzer/mysql/worker/perfschema/worker.go:409]: I changed this part. I don't think it makes sense now to check if `prevClass` has DigestText as you can simply take it directly from `row`.

**Comment by percona-csalguero** [comment]
In my opinion it should be merged.
Perf schema tables don't have indexes so reading the same info twice, using a where condition without indexes, forces MySQL to read all rows and filter the result. That generates a lot of extra load.

**Review by arvenil — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Comment by arvenil** [comment]
Resolves also PMM-1144

**Pull request merged**